Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -936,17 +936,17 @@
 "settings-support" = "Support Wikipedia";
 "settings-title" = "Settings";
 "share-a-fact-made-with" = "Made with the Wikipedia app";
-"share-article-name-on-wikipedia" = "\"$1\" on @Wikipedia:";
+"share-article-name-on-wikipedia" = "\"$1\" on Wikipedia:";
```

Comment:
I'm trying to figure out why my branch carried these changes and why it wasn't on main already

Reply:
@mazevedofs Looking at the originating PR https://github.com/wikimedia/wikipedia-ios/pull/4192, it looks like while the localized string updates in code were committed, the contributor did not also commit the `Localizable.strings` file updates. Because of that, on this branch (or any other branch based off current `main`) this commit change will show during project build because of the localization build scripts.

Reply:
Thanks Deepak! I hadn't noticed that :)